app/elements/pages/page-browse.html

```diff
@@ -290,6 +291,9 @@ <h2 class="flex"><span>[[pageTitle]]</span> <span class="count">[[_elementCount]
       cartOpen: function(e) {
         this.fire('cart-open');
       },
+      scrollTop: function() {
```

Comment:
This overrides the `scrollTop` property.  It should be changed to something else like `scrollToTop`